### ISSUE TITLE
Negative loss value in multi_task_loss

### ISSUE THREAD

**Issue by caki35** [ywatanabe1989/custom_losses_pytorch#4]
Hello. First of all, thank you for your implementation. I'd like to ask why I keep obtaining negative loss value due to std lower than 1 and thus + torch.log(stds) term. Is it related to my dataset which has class imbalance data(very small foreground aginst the much bigger background) or the multi-loss definitions? What do you suggest about this problem?

**Comment by ywatanabe1989** [comment]
I suppose it is a possible case. Is it a problem that loss values are negative? I mean regardless of the sign of loss, metrics such as accuracy can increase. Please consider validating metrics and let me know if something is wrong with your case.

**Comment by caki35** [comment]
I carried out some experiments with different seeds. Actually, there is no problem with the learning of the model. Accuracy reached 92 dice scores. However basic summation of the losses can also achieve this level. Training using multiloss did not outperform training using the summation of losses in my case.
